src/main/java/com/ollymonger/dynmap/portals/DynmapPortalsPlugin.java

```diff
@@ -126,20 +127,34 @@ public void onPortalCreate(PortalCreateEvent event) throws IOException {
     }
 
     private void writePortals() {
-        try{
+        try {
             Path path = Paths.get(getDataFolder().getAbsolutePath(), "portals.json");
             File file = new File(path.toString());
             file.getParentFile().mkdirs();
             OutputStreamWriter os = new OutputStreamWriter(new FileOutputStream(file), "UTF-8");
 
             os.write(gson.toJson(this.registeredPortals, registeredPortalListType));
+            getLogger().info("portal done");
 
             os.close();
         } catch (IOException e) {
             e.printStackTrace();
         }
     }
 
+    private void loadPortals() {
+        try {
+            Path path = Paths.get(getDataFolder().getAbsolutePath(), "portals.json");
```

Comment:
This code is now in two places. It might be good to refactor this in future to a single `getPortalsFile` method